Solutions/Corvus.Monitoring.lutconfig

```diff
@@ -0,0 +1,6 @@
+<LUTConfig Version="1.0">
```

Comment:
What's this file?

Reply:
Visual Studio Live Unit Testing config file.

According to the docs, these should be checked in. No big deal if you want rid of it though.